File: testprojects/master_scala_main/pipelines/ScalaGCP/code/src/main/scala/graph/everythingSG_1/RowDistributor_1.scala
package graph.everythingSG_1

import io.prophecy.libs._
import config.ConfigStore._
import udfs.UDFs._
import udfs._
import org.apache.spark._
import org.apache.spark.sql._
import org.apache.spark.sql.functions._
import org.apache.spark.sql.types._
import java.time._

object RowDistributor_1 {

  def apply(spark: SparkSession, in: DataFrame): (DataFrame, DataFrame) =
    (in.filter(col("`c- short`") > lit(-10)),
     in.filter(col("`c  - int`") > lit(-100))
    )

}

File: testprojects/master_python_mainone/pipelines/StreamingAllPipeline/code/streamingallpipeline/streaming/main/one/this/graph/RowDistributor_1.py
from pyspark.sql import *
from pyspark.sql.functions import *
from pyspark.sql.types import *
from prophecy.libs import typed_lit
from streamingallpipeline.streaming.main.one.this.config.ConfigStore import *
from streamingallpipeline.streaming.main.one.this.udfs.UDFs import *

def RowDistributor_1(spark: SparkSession, in0: DataFrame) -> (DataFrame, DataFrame):
    df1 = in0.filter((col("c_tinyint") > lit(-100)))
    df2 = in0.filter((col("c_int") > lit(-100)))

    return df1, df2
